tests/shutdown/dummy.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+exit 0
```

Comment:
we already have `exit0.c` in the repo root and it's exposed as a cc_binary, perhaps also expose it as a cc_test so we can avoid the bash dep here?